src/main/scala/mesosphere/marathon/core/launchqueue/impl/ReviveOffersStreamLogic.scala

```diff
@@ -72,11 +72,9 @@ object ReviveOffersStreamLogic extends StrictLogging {
 
     reviveStateFromInstancesAndDelays(defaultRole)
       .buffer(1, OverflowStrategy.dropHead) // While we are back-pressured, we drop older interim frames
-      .throttle(1, minReviveOffersInterval)
+      .via(RateLimiterFlow.apply(minReviveOffersInterval))
       .map(_.roleReviveVersions)
-      .map(l => { logger.info(s"roleReviveVersions = ${l}"); l })
       .via(reviveDirectiveFlow(enableSuppress))
```

Comment:
This was very helpful while debugging. It is a relatively low log overhead so let's get it back 🙏 :
```suggestion
      .via(reviveDirectiveFlow(enableSuppress))
      .map(l => { logger.info(s"Issuing following suppress/revive directives: = ${l}"); l })
```